### PR TITLE
Add self-test operation of the GPU container when the GPU container starts

### PR DESCRIPTION
**Pull request by Mingbo-Lee**
I add self-test operation of the GPU container when the GPU container starts so that the error can be found quickly if a container could not run correctly.

### CODE REVIEW

**Comment by Mingbo-Lee** [comment]
Could you please TBR? @zhouaihui

File: docker/secretflow_entrypoint.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+   _____                     __  ________             
+  / ___/___  _____________  / /_/ ____/ /___ _      __
+  \__ \/ _ \/ ___/ ___/ _ \/ __/ /_  / / __ \ | /| / /
+ ___/ /  __/ /__/ /  /  __/ /_/ __/ / / /_/ / |/ |/ / 
+/____/\___/\___/_/   \___/\__/_/   /_/\____/|__/|__/  
+"""
+print(banner)
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+import secretflow
```
Comment (zhouaihui) [inline docker/secretflow_entrypoint.py:12]: Can you sort the imports please?

File: docker/secretflow_entrypoint.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import logging
+import os
+import subprocess
+import sys 
+import re
```
Comment (zhouaihui) [inline docker/secretflow_entrypoint.py:5]: Please check again, `import re` should be in the front of `import subsprocess`.

And [isort](https://pycqa.github.io/isort/) might be helpful.

Comment (Mingbo-Lee) [reply docker/secretflow_entrypoint.py]: I apologize for my negligence. I fix the issue in newst commit.Please you review again.

**Review by zhouaihui — APPROVED** [review]
LGTM

**Pull request merged**